database/seedMethods.js

```diff
@@ -2,14 +2,15 @@
 /* eslint-disable no-shadow */
 
 const Promise = require('bluebird');
```

Comment:
Have you looked into async await? Highly recommend

Reply:
+1 on Brian's comment. Though I recall using bluebird was how Andrew wrote the code. If everything is working fine and to db query time requirements then perhaps leave it for now.